pyfactorybridge/__init__.py

```diff
@@ -167,8 +197,35 @@ def load_game(self, SaveName, EnableAdvancedGameSettings) -> dict[str, str]:
             },
         )
 
-    def upload_save_game(self, data) -> None:
-        raise NotImplementedError
+    def upload_save_game(
+        self,
+        save_file_path: str,
```

Comment:
Could this be renamed to `SaveFilePath` to keep it consistent?

Reply:
Sure, I thought you only named variables directly passed to the API in PascalCase and used snake_case for the rest. I mean all other parameters are also starting in lower case.

But if you insist, I can change that :)

Reply:
Ah, no, good point - that's fine, we can keep it like that